src/genbenchQC/report/report_generator.py

```
@@ -105,24 +105,24 @@ def generate_train_test_html_report(clusters, train_filename, train_seq, test_fi
     with open(output_path, 'w') as file:
         file.write(template)
 
-def generate_dataset_html_report(stats1, stats2, results, output_path, plots_path, threshold, end_position, plot_type):
+def generate_dataset_html_report(stats1, stats2, output_path, plots_path, end_position, plot_type, results=None, threshold=None):
```

Ruff (0.14.4)

Unused function argument: `results`

(ARG001)